crates/exex/exex/src/manager.rs

```diff
@@ -25,6 +25,12 @@ use tokio::sync::{
 };
 use tokio_util::sync::{PollSendError, PollSender, ReusableBoxFuture};
 
+/// Default max size of the internal state notifications buffer
+/// 1024 notifications in the buffer is 3.5 hours of mainnet blocks,
+/// or 17 minutes of 1-second blocks. It should be enough for any use-case,
+/// and we can make it configurable once there's a demand for that.
```

Comment:
nit: some punctuation to delineate the sentences, and removal of a feature idea/demand thing, i don't think this fits in public docs

```suggestion
/// Default max size of the internal state notifications buffer.
///
/// 1024 notifications in the buffer is 3.5 hours of mainnet blocks,
/// or 17 minutes of 1-second blocks.
```